server/src/helpers/standupValidation.ts

```diff
@@ -0,0 +1,10 @@
+const { body, check } = require("express-validator");
+
+export const validateStandup = [
+  body("title").not().isEmpty(),
```

Comment:
We can also send custom error message:
https://express-validator.github.io/docs/validation-chain-api.html#notempty
https://express-validator.github.io/docs/validation-chain-api.html#notempty

```suggestion
  body("title").trim().notEmpty().withMessage('Title should not be empty'),
```

Reply:
Yes, custom messages can be added